.github/workflows/test.yml

```diff
@@ -0,0 +1,54 @@
+name: "Run unit tests"
+
+on:
+  - push
+  - pull_request
+
+env:
+  COMPOSER_MEMORY_LIMIT: -1
+
+jobs:
+  test:
+    name: "Build"
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 12
+      matrix:
+        php: ['7.4', '8.0', '8.1']
```

Comment:
Current PHP requirement is 7.1 and higher. Though those are no longer actively developed, as long as we haven't raised the minimum requirement we should run on those versions as well.

Reply:
You're right. I copied those versions from my fav gh action file. I'll include 7.1+.